android/app/src/main/java/com/example/medcarapp/ManualControl.java

```diff
@@ -50,13 +52,13 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onMove(int angle, int strength) {
                 int adjustedAngle = adjustAngle(angle);
                 int adjustedSpeed = adjustSpeed(strength, angle);
+                int speed = speedometerSpeed(strength,angle);
```

Comment:
This variable name could be changed to speedometerSpeed for improved readability.

Reply:
That's true, changed it.